packages/controllers/package.json

```diff
@@ -17,7 +17,6 @@
     "posttest": "jest-it-up",
     "test:ci": "yarn test:prepare && ./scripts/test-ci.sh",
     "build:pre-tsc": "yarn build:ajv && node scripts/transpileSchemaTypes.js",
-    "build:post-tsc": "echo 'N/A'",
```

Comment:
I removed all scripts with `echo N/A`, since they're not required for Yarn 3 workspaces `foreach`.